docs/cookbook/Translations with SiteAccess

```diff
@@ -0,0 +1,119 @@
+## Description
+
+Being new to eZ and combining the translated content with multiple site access successfully can be a little tricky to get you head around.
```

Comment:
Please, could you:
eZ to replace by _eZ Platform_
site access need First Uppercase notation